src/watchers/indexable-author.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Author watcher to save the meta data to an Indexable.
+ *
+ * @package Yoast\YoastSEO\Watchers
+ */
+
+namespace Yoast\YoastSEO\Watchers;
+
+use Yoast\YoastSEO\Exceptions\No_Indexable_Found;
+use Yoast\YoastSEO\Formatters\Indexable_Author as Indexable_Author_Formatter;
+use Yoast\YoastSEO\WordPress\Integration;
+use Yoast\YoastSEO\Models\Indexable;
+
+/**
+ * Watches an Author to save the meta information when updated.
+ */
+class Indexable_Author implements Integration {
```

Comment:
Once the below points have been applied it will likely be possible to create an abstract `Indexable_Watcher` class that all these inherit from which implements the majority of functions.

The `delete_meta` function is already mostly the same for all the watchers I believe.